backend/src/main/java/com/woowacourse/friendogly/image/ImageController.java

```diff
@@ -0,0 +1,25 @@
+package com.woowacourse.friendogly.image;
+
+import com.woowacourse.friendogly.infra.FileStorageManager;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+public class ImageController {
+
+    private final FileStorageManager fileStorageManager;
+
+    public ImageController(FileStorageManager fileStorageManager) {
+        this.fileStorageManager = fileStorageManager;
+    }
+
+
+    @PostMapping("/test-upload")
+    public ResponseEntity<String> uploadFile(@RequestParam("file") MultipartFile file) {
+        String imageUrl = fileStorageManager.uploadFile(file);
+        return ResponseEntity.ok(imageUrl);
+    }
```

Comment:
```suggestion
    public ImageController(FileStorageManager fileStorageManager) {
        this.fileStorageManager = fileStorageManager;
    }

    @PostMapping("/test-upload")
    public ResponseEntity<String> uploadFile(@RequestParam("file") MultipartFile file) {
        String imageUrl = fileStorageManager.uploadFile(file);
        return ResponseEntity.ok(imageUrl);
    }
```

개행이 2줄이군요!!
예시 코드는 나중에 까먹지 않고 삭제할 수 있도록 TODO를 남겨도 좋을 것 같아요!